README.md

```diff
@@ -1,70 +1,57 @@
+# NASA Imagery
+![](https://i.imgur.com/RdcsHKt.png)
+![](https://i.imgur.com/AcV9JkQ.png)
 
-## Available Scripts
+> In this app, the user can choose from a list of celestial bodies (planets, black holes, suns, galaxies and nebulas) and see images directly from NASA. The images are shown with their title and the date they were created on.
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- HTML, CSS, JS
+- Git, GitHub, Webpack
+- React, Redux
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[Live Demo](https://daf98-nasa-images.herokuapp.com)
+[Video Demo](https://www.loom.com/share/651329c7ce704382b54b8146b70ca530)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### To launch the project locally:
+#### Step 1:
+- Clone this repo locally using git in the command line and the following command
+ ```bash
+ > git clone https://github.com/Daf98/NASA-Imagery.git
+ ```
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+#### Step 2:
+- In the terminal, you can run:
+```bash
+> cd NASA-Imagery
+> npm install
+> npm start
+```
+- Your page will start on localhost:3000.
+## Author
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+👤 **Dafne Azzolina**
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- GitHub: [@Daf98](https://github.com/Daf98)
+- Twitter: [@dafne_azzolina](https://twitter.com/dafne_azzolina)
+- LinkedIn: [dafne-azzolina](https://www.linkedin.com/in/dafne-azzolina/)
 
-### `npm run eject`
+## Show your support
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+Give a ⭐️ if you like this project!
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Acknowledgments
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- Origin design by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
```

Comment:
Well done giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 🤓

Reply:
Thank you :)